src/packages/paths.py

```
@@ -0,0 +1,24 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)